allennlp/modules/elmo_lstm.py

```diff
@@ -42,6 +42,8 @@ class ElmoLstm(_EncoderBase):
         :class:`~allennlp.modules.lstm_cell_with_projection.LstmCellWithProjection`.
     num_layers : ``int``, required
         The number of bidirectional LSTMs to use.
+    require_grad: ``bool``, optional
+        Compute gradients of model parameters?
```

Comment:
"If `True`, computes gradient of ELMo parameters, which is useful for fine tuning" or something.